README.md

```diff
@@ -1,3 +1,280 @@
-Placeholder for possible webpack loader for Twitter Bootstrap
+# bootstrap-loader
 
-Considering changing the name from bootstrap-sass-loader.
+Load Bootstrap styles and scripts in your Webpack bundle. This loader uses SASS to process CSS styles. Bootstrap 3 & 4 are supported.
+
+## Installation
+
+Get it via npm:
+
+```bash
+npm install bootstrap-loader
+```
+
+Don't forget to install these dependencies:
+
+```bash
+# Bootstrap 3
+npm install bootstrap-sass
+
+# or Bootstrap 4
+npm install twbs/bootstrap#v4.0.0-alpha
+
+# Node SASS & other loaders needed to handle styles
+npm install css-loader node-sass resolve-url-loader sass-loader style-loader
+```
+
+## Usage
+
+Simply require it:
+
+```js
+require('bootstrap-loader');
+```
+
+Or add `bootstrap-loader` as an entry point in your webpack config:
+
+```js
+entry: [ 'bootstrap-loader', './app' ]
+```
+
+Config is optional. It can be placed in root dir with name `.bootstraprc`. You can write it in `YAML` or `JSON` formats.
```

Comment:
See this file <BLAH> for the default config file. Note, we don't recommend this as upgrades might make changes that will break your app.